src/components/utilities.js

```diff
@@ -28,12 +28,34 @@ export const Utilities = {
   },
 
   async getSupportedVersions() {
-    try {
-      const resp = await fetch(process.env.SUPPORTED_VERSIONS_JSON);
-      const data = await resp.json();
-      return { ...data.supportedVersions };
-    } catch (error) {
-      throw error;
-    }
+    const internalAddonVersions = fetch(process.env.SUPPORTED_VERSIONS_JSON)
+        .then(response => response.json())
+        .then(response => response.supportedVersions);
+
+    const externalAddonVersions = fetch("https://kurl-sh.s3.amazonaws.com/external/addon-registry.json")
```

Comment:
should this URL be configurable like the internal versions file?

Reply:
Right now all environments use the same url so I didn't bother making it configurable